code/__HELPERS/paths/path.dm

```diff
@@ -281,7 +281,7 @@
 	/// Are we being thrown?
 	var/thrown = FALSE
 	/// Are we anchored
-	var/anchored = FLASH_LIGHT_POWER
```

Comment:
LMAO WHAT

Reply:
listen ok I was relying on FALSE to autocorrect and I don't think it really did